polyaxon_cli/cli/run.py

```diff
@@ -135,6 +137,9 @@ def run(ctx,
     tags = validate_tags(tags)
 
     if local:
+        if ctx.obj["offline"]:
+            os.environ['POLYAXON_IS_OFFLINE'] = 'true'
```

Comment:
Does this get reverted from the `environ` when the user does not pass the offline flag? If not then it's better no to wither:
 * pass the flag to other components that require it
 * rely entirely on the user's env var from the start